docs/topics/request-response.rst

```diff
@@ -525,11 +525,11 @@ Response objects
        (for single valued headers) or lists (for multi-valued headers).
     :type headers: dict
 
-    :param body: the response body. It must be str, not unicode, unless you're
-       using a encoding-aware :ref:`Response subclass
+    :param body: the response body. To access the decoded text as str you can
```

Comment:
`str` is not right in Python 2; what do you think about adding something like "decoded text as str (unicode in Python 2)"?

Reply:
Python 2?  What's that? ha 
Sure thing.